src/modules/github/reducer.ts

```diff
@@ -1,36 +1,44 @@
-import { SEARCH_GITHUB_REPOSITORIES_REQUEST, SEARCH_GITHUB_REPOSITORIES_SUCCESS,SEARCH_GITHUB_REPOSITORIES_ERROR } from './action';
+import {
+    SEARCH_GITHUB_REPOSITORIES_REQUEST,
+    SEARCH_GITHUB_REPOSITORIES_SUCCESS,
+    SEARCH_GITHUB_REPOSITORIES_ERROR,
+} from './action';
 import { AnyAction } from 'redux';
 import { GithubRepositoryItem } from './types';
+import { act } from 'react-dom/test-utils';
 
 export interface GithubRepositoryState {
     list: GithubRepositoryItem[],
     error?: Error,
-    isLoading: boolean
+    isLoading: boolean,
+    total_repositories_count: number
```

Comment:
Please use `camelCase` in JS and TS code instead of `snake_case`.
```suggestion
    totalRepositoriesCount: number,
```